README.md

```diff
@@ -21,9 +15,9 @@ You'll get:
 
 ## Demo
 
-### [Blog](https://nextjs-blog.sanity.build)
-
-### [Studio](https://nextjs-blog.sanity.build/studio)
+| [Blog](https://nextjs-blog.sanity.build)                              | [Studio](https://nextjs-blog.sanity.build/studio)                                |
+| --------------------------------------------------------------------- | -------------------------------------------------------------------------------- |
+| [![blog](./assets/readme/blog.png)](https://nextjs-blog.sanity.build) | [![studio](./assets/readme/studio.png)](https://nextjs-blog.sanity.build/studio) |
```

Comment:
The most important part is to make sure the visuals are consistent and use the same pictures as previews on [Vercel.com](http://vercel.com/). I would also refrain from using dark mode, it stands out a bit too much

Reply:
Will be sure to keep this in mind.
Do we know who is doing the content / when it's finished? Is it hidde?